source/releasenotes/about.rst

```diff
@@ -17,7 +17,20 @@
 What's New in |release|
 =======================
 
-Apache CloudStack |release| is a 4.19 LTS minor release with over 170 fixes
+Apache CloudStack 4.19.3 is a minor release with a limited number of
+assorted fixes.
+
+* Fixes broken console access after upgrade to 4.19.2.0
+* Improve listing of Vmware Datacenter VMs for migration to KVM
+* Infinite scroll UI component to retrieve more items on reaching end of list
+* Prevention of duplication HA jobs and alerts
+* Fix SAML2 plugin limitations and SAML multi-account selector in the UI
+* Improvements to Linstor
+
+What's New in 4.19.2
+====================
+
+Apache CloudStack 4.19.2 is a 4.19 LTS minor release with over 170 fixes
```

Comment:
```suggestion
Apache CloudStack 4.19.3 is a 4.19 LTS minor release with over 170 fixes
```

Reply:
the count of fixes needs to be verified.

Reply:
this count hasn’t changed. it is for 4.19.2. 4.19.3 is above and has no count